### PR TITLE
Add Stored Credentials Support to Okta

### PR DESCRIPTION
**Pull request by MichaelYochpaz**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4767)

## Description
Add support to stored credentials (type 9 field) for Okta integrations.

### CODE REVIEW

File: Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 
+import urllib3
 # noqa: F401
 # noqa: F401
 # noqa: F401
 # noqa: F401
```
Comment (bziser) [inline Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py:8]: I don't think we need it 4 times

File: Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py [hunk 2]
```diff
@@ -918,7 +919,11 @@ def main():
     user_profile = None
     params = demisto.params()
     base_url = urljoin(params['url'].strip('/'), '/api/v1/')
-    token = params.get('apitoken')
+    token = params.get('credentials', {}).get('password') or params.get('apitoken')
```
Comment (bziser) [inline Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py:922]:
```suggestion
    token = params.get('credentials', {}).get('password', '') or params.get('apitoken', '')
```

File: Packs/Okta/Integrations/Okta_IAM/Okta_IAM.yml [hunk 3]
```diff
@@ -7,10 +7,16 @@ configuration:
   name: url
   required: true
   type: 0
-- display: API Token (see Detailed Instructions)
+- name: credentials
+  required: false
```
Comment (bziser) [inline Packs/Okta/Integrations/Okta_IAM/Okta_IAM.yml:11]:
```suggestion
```

File: Packs/Okta/Integrations/Okta_IAM/Okta_IAM.yml [hunk 4]
```diff
@@ -7,10 +7,16 @@ configuration:
...
+  type: 9
+  displaypassword: API Token (see detailed instructions)
+  hiddenusername: true
+- display: API Token (See detailed instructions)
   name: apitoken
-  required: true
+  required: false
```
Comment (bziser) [inline Packs/Okta/Integrations/Okta_IAM/Okta_IAM.yml:17]:
```suggestion
```

File: Packs/Okta/Integrations/Okta_v2/Okta_v2.yml [hunk 5]
```diff
@@ -7,10 +7,16 @@ configuration:
   name: url
   required: true
   type: 0
-- display: API Token (see Detailed Instructions)
+- name: credentials
+  required: false
```
Comment (bziser) [inline Packs/Okta/Integrations/Okta_v2/Okta_v2.yml:11]:
```suggestion
```

File: Packs/Okta/Integrations/Okta_v2/Okta_v2.yml [hunk 6]
```diff
@@ -7,10 +7,16 @@ configuration:
...
+  type: 9
+  displaypassword: API Token (see detailed instructions)
+  hiddenusername: true
+- display: API Token (see detailed instructions)
   name: apitoken
-  required: true
+  required: false
```
Comment (bziser) [inline Packs/Okta/Integrations/Okta_v2/Okta_v2.yml:17]:
```suggestion
```

**Review by bziser — COMMENTED** [review]
👍

File: Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py (hunk 1 above)
Comment (MichaelYochpaz) [reply Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py:8]:
```suggestion
```

**Review by bziser — APPROVED** [review]

**Pull request merged**